### PR TITLE
Removed Topic API from the specification

### PR DESCRIPTION
**Pull request by valgog**
* Simplified the API by removing the Topic API compleatly
* Made changes to address comments from the code review

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/78" target="_blank">ReviewNinja

File: api/nakadi-event-bus-api.yaml [hunk 1]
```diff
@@ -414,20 +420,31 @@ paths:
...
-          description: Array of cursors indicating the partitions to read from and the starting point on
-            each. Assumes the offset on each cursor is not inclusive (i.e., first delivered Event is the
+          description: |
+            Array of cursors indicating the partitions to read from and the starting offset on each such a
+            partition.
+
+            Assumes the offset on each cursor is not inclusive (i.e., first delivered Event is the
             **first one after** the one pointed to in the cursor).
+
+            If the header is not present, the stream will start from the newest event available in the
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: I think not from the newest, but from the one that will come after the newest.

File: api/nakadi-event-bus-api.yaml [hunk 2]
```diff
@@ -463,11 +480,10 @@ paths:
             Maximum time in seconds to wait for the flushing of each chunk (per partition).
 
             * If the amount of buffered Events reaches `batch_limit` before this batch_flush_timeout is
-            reached, the messages are immediately flushed to the client.
+            reached, the messages are immediately flushed to the client and batch flush timer reset.
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: *resets (or "is reset")

File: api/nakadi-event-bus-api.yaml [hunk 3]
```diff
@@ -528,51 +541,17 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
-  /topics:
+  /event-types/{name}/partitions':
```
Comment (v-stepanov) [inline api/nakadi-event-bus-api.yaml]: So we decided to keep endpoints exposing partitions info ?

File: api/nakadi-event-bus-api.yaml [hunk 4]
```diff
@@ -29,11 +29,12 @@ info:
     before persisting it. Usually enrichment affects the metadata of an Event but is not limited to.
```
Comment (tfrauenstein) [inline api/nakadi-event-bus-api.yaml]: Generally:  The API now looks much more consistent and has profited by uniform usage of EventType and removing the Topic resource aspects. 
Pls. find some new comments below. 
Pls. also take care that all other agreed comments of the former review will go into the next API version.

File: api/nakadi-event-bus-api.yaml (hunk 3 above)
Comment (valgog) [inline api/nakadi-event-bus-api.yaml]: We have to. At least for the monitoring purposes.

Comment (valgog) [inline api/nakadi-event-bus-api.yaml]: !resolved

**Pull request merged**